OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum ReferencePoints { TopLeft, BottomRight }
```

Comment:
A bitfield (with the `[Flags]` attribute) would be better than a pair of points.